pipelines/common/src/test/java/com/google/fhir/analytics/FhirStoreUtilTest.java

```diff
@@ -31,15 +31,15 @@
 import org.hamcrest.Matchers;
 import org.hl7.fhir.r4.model.Bundle;
 import org.hl7.fhir.r4.model.Patient;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.Answers;
 import org.mockito.ArgumentMatchers;
 import org.mockito.Mock;
-import org.mockito.junit.MockitoJUnitRunner;
+import org.mockito.junit.jupiter.MockitoExtension;
 
-@RunWith(MockitoJUnitRunner.class)
+@ExtendWith(MockitoExtension.class)
```

Comment:
Can we target the migration of one complete maven module in a PR, i.e. all the classes in the `common` module can be migrated at once.

Reply:
@chandrashekar-s  I have migrated all classes in common module to junit5

Reply:
@meetmehta1198 I don't see any new changes being committed to the PR, can you please have a look?

Reply:
@chandrashekar-s Can you please check now? I have pushed the changes. Apologies for the delay!